### ISSUE TITLE
Parse Square brackets

### ISSUE THREAD

**Issue by captn3m0** [captn3m0/youtube-cue#64]
Sample: https://youtu.be/Yir4WyDi8rU

```
[00:00:00] 1. Steve Kroeger x Skye Holland - Through The Dark 
[00:02:53] 2. Gabri Ponte x Jerome - Lonely 
[00:05:55] 3. Sander van Doorn x Harris & Ford - Spotlight 
[00:08:49] 4. Mariana BO & LaCrème - Red Eagle 
[00:11:48] 5. TJR - Shake That 
```

**Comment by porg** [comment]
Ok, seems that I provided a new syntax specimen, which you want to support! Nice! Thanks!

**Comment by captn3m0** [comment]
Released 1.0.4 with the fix.

**Comment by porg** [comment]
I hereby confirm: Updated to 1.0.4. Tried sample URL. Got cue sheet with artist and title being parsed correctly.
